pkg/util/conn_pool.go

```diff
@@ -45,7 +46,7 @@ type ConnPool struct {
 	conns map[string]*connEntry
 }
 
-// Create a new ConnPool instance and start the garbage collector running
+// NewConnPool create a new connection pool instance and start the garbage collector running
```

Comment:
Grammar nit: "creates" not "create", "starts" not "start"

Reply:
Unless github is caching something it still looks like "create" to me. Should be "creates".